Almacenar varios votos

marjimveg1 opened the pull request:
**¿Que hace este pull request?**
Se ha añadido la funcionalidad de almacenar varios votos a la vez.

**Incidencia**: #20 https://github.com/danhidsan/decide-ganimedes-almacenamiento/issues/20

**¿Por que estas haciendo el pull request?. Contexto del pull request.**
Desde el sub-sistema cabina se pidió que se implementara dicha funcionalidad.

**¿Por dónde debe empezar el revisor?**
Estar localizado en la rama almacenar_varios_votos haciendo uso del comando:
$ git checkout almacenar_varios_votos

El archivo a comprobar se encuentra dentro del módulo store, en el archivo views.py

marjimveg1 committed:
feat
añadido almacenar varios votos

danhidsan commented on decide/store/views.py:
```diff
@@ -58,15 +58,21 @@ def post(self, request):
-        v.a = a
-        v.b = b
+        # Recorro los votos
+        for v in vote:
```
Aquí no se está recorriendo una array de votos, estas recorriendo un único voto

danhidsan commented on decide/store/views.py:
```diff
@@ -58,15 +58,21 @@ def post(self, request):
+        # Recorro los votos
+        for v in vote:
+            # Cojo el voto
+            a = v.get("a")
```
Al estar recorriendo un único voto, estas lineas de código no son correctas.

danhidsan reviewed (changes requested):
Los cambios no funcionan. Se ocasiona un "Internal server error", lo que quiere decir que hay un fallo en la API de almacenamiento. 

Añado la traza del error en una imagen

<img width="713" alt="captura de pantalla 2018-12-23 a las 19 34 11" src="https://user-images.githubusercontent.com/25649572/50386617-add45b00-06ea-11e9-9c90-527790203d7e.png">

danhidsan committed:
feat: Solucionado error en el código al guardar votos de forma múltiple

    - Se ha cambiado el código para solucionar un error que hacía que no se pudiera votar con multipl      es votos. Este fallo fue ocasionado al incluir la funcionalidad.
    - Se han añadido mensajes en las respuestas del servidor en la ruta de almacenamiento de votos

 Incidencia: #20

danhidsan committed:
Resueltos conflictos con master desde 'almacenar_varios_votos'

Incidencia: #20

danhidsan committed:
feat: Añadido cambios en el javascript de votación para la votación múltiple

    - Se ha hecho un cambio en el java script incrustado en booth.html, para que funcione la votación      múltiple

 Incidencia: #20

danhidsan committed:
test: Cambios en los test de almacenamiento de votos debido al voto multiple

    - Se han hecho los cambios pertinentes provacados al añadir la nueva funcionalidad de voto multiple

 Incidencia: #20

danhidsan committed:
test: Cambios en los test de votación debido a la nueva votación multiple

 Incidencia: #20
